includes/class-webmention-receiver.php

```diff
@@ -283,8 +292,8 @@ public static function check_dupes( $commentdata ) {
 			$commentdata['comment_ID'] = $comment->comment_ID;
 			$commentdata['comment_approved'] = $comment->comment_approved;
 		}
-
-		return $commentdata;
+		// Add hook for Semantic Linkbacks to Add Duplicate Comment Data
+		return apply_filters( 'webmention_check_dupes', $commentdata );
 	}
```

Comment:
The idea of this is that for webmentions, Semantic Linkbacks can link directly into the duplicate webmention filter. There is no point in checking this for Pingbacks and Trackbacks, because they both stop if they find a duplicate. It is part of their specification.